src/test/java/org/folio/entitlement/utils/WireMockUtil.java

```diff
@@ -0,0 +1,60 @@
+package org.folio.entitlement.utils;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.any;
+import static com.github.tomakehurst.wiremock.client.WireMock.delete;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.put;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.client.MappingBuilder;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.matching.UrlPattern;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class WireMockUtil {
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+  public static void stubGet(WireMock wireMockClient, int priority, UrlPattern urlPattern, Object response)
+    throws Exception {
+    stubGet(wireMockClient, priority, urlPattern, response, 200);
+  }
+
+  public static void stubGet(WireMock wireMockClient, int priority, UrlPattern urlPattern, Object response,
```

Comment:
methods usually start with a verb according to 
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

In Eureka made an exception for stubs in tests, that can be named as object

Reply:
To stub is a verb (stubbing is a common term)